python/examples/README.md

```diff
@@ -0,0 +1,357 @@
+A 🥕 was waiting for him on the table.
+He will put on his 👖 and get out of the 🏠 for a walk.
+```
+
+Passing an environment variable with WasmEdge is similar
+```shell-session
+wasmedge \
+  --env PYTHONPATH=/external-packages \
+  --dir /external-packages:$PWD/tmp/venv-emoji/lib/python3.11/site-packages \
+  --dir workdir:$PWD/workdir \
+  tmp/python-3.11.1-wasmedge.wasm \
+  workdir/emojize_text.py workdir/source_text.txt
+...
+```
+
+## Running the Docker container
+
+Docker+WASM uses the WasmEdge runtime internally. To leverage it we have packaged the python-3.11.1-wasmedge.wasm binary in a container image available as `ghcr.io/vmware-labs/python-wasm:3.11.1-wasmedge`.
+
+Here is an example of running the Python repl from this container image. As you can see from the output of the interactive session, the container includes only `python.wasm` and the standard libraries from `usr`. No base OS images, no extra environment variables, or any other clutter.
+
+```shell-session
+docker run --rm \
+  -i \
+  --runtime=io.containerd.wasmedge.v1 \
+  --platform=wasm32/wasi \
+  ghcr.io/vmware-labs/python-wasm:3.11.1-wasmedge \
+  -i
+
+Python 3.11.1 (tags/v3.11.1:a7a450f, Feb 17 2023, 11:01:02) ...  on wasi
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import sys
+>>>
+>>> sys.platform
+>>> 'wasi'
+>>>
+>>> import os
+>>> os.listdir('.')
+['python.wasm', 'etc', 'usr']
+>>>
+>>> [k for k in os.environ.keys()]
+['PATH', 'HOSTNAME']
+```
+
+You can also run the Docker container to execute a one-liner like this.
+
+```shell-session
+docker run --rm \
+  --runtime=io.containerd.wasmedge.v1 \
+  --platform=wasm32/wasi \
+  ghcr.io/vmware-labs/python-wasm:3.11.1-wasmedge \
+  -c "import os; print([k for k in os.environ.keys()])"
+
+['PATH', 'HOSTNAME']
+```
+
+## Running the Docker container with dependencies
+
+The `python-wasm` container image comes by default just with the Python standard library, so if your project has extra dependencies you will need to take care of them. Let's reuse the `venv-emoji` environment in which we installed `emoji` in [the example above](#using-a-virtual-environment).
+
+We need to do three things
+
+1. Ensure that the `emoji` module installed in the `venv-emoji` folder is mounted in the running `python-wasm` container
+2. Ensure that it is also on the `PYTHONPATH` within the running `python-wasm` container
+3. Ensure that the python program and its data (in this case `workdir/emojize_text.py` and `workdir/source_text.txt`) are also mounted in the container
+
+A vital piece of knowledge here is that whatever you mount in the running container gets automatically pre-opened by the WasmEdge runtime. Same goes for all environment variables that you pass to the container when you run it. 
+
+One way of doing what we want is to just mount `site-packages` from `venv-emoji` over the `site-packages` folder of the pre-compiled path in `/usr/local`. This is what this could look like:
```

Comment:
```suggestion
One way of doing what we want is to just mount `site-packages` from `venv-emoji` over the `site-packages` folder of the pre-compiled path in `/usr/local`. This is how this could look like:
```